packages/components/src/internal/components/samples/utils.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { List } from 'immutable';
+
+import { App, getOmittedSampleTypeColumns } from '../../../index';
+import { isFreezerManagementEnabled } from '../../app/utils';
+
+// Duplicated from sampleManagement/packages/workflow/src/constants.ts
+export const CHECKED_OUT_BY_FIELD = 'checkedOutBy';
+export const INVENTORY_COLS = [
```

Comment:
Do we want to opt for hoisting these two constants entirely to ui-components?

Reply:
Perhaps they should be moved to @labkey/freezermanager, now that LKB uses @labkey/freezermanager

Reply:
Done!